scripts/linuxaarch64/arch-bootstrap.sh

```diff
@@ -0,0 +1,237 @@
+)
+BASIC_PACKAGE_ARMS=(${PACMAN_PACKAGES_ARM[*]} filesystem)
+
+stderr() {
+  echo "$@" >&2
+}
+
+debug() {
+  stderr "--- $@"
+}
+
+extract_href() {
+  sed -n '/<a / s/^.*<a [^>]*href="\([^\"]*\)".*$/\1/p'
+}
+
+fetch() {
+  curl -L -s "$@"
+}
+
+uncompress() {
+  local FILEPATH=$1 DEST=$2
+
+  case "$FILEPATH" in
+    *.gz) tar xzf "$FILEPATH" -C "$DEST";;
+    *.xz) xz -dc "$FILEPATH" | tar x -C "$DEST";;
+    *) debug "Error: unknown package format: $FILEPATH"
+       return 1;;
+  esac
+}
+
+###
+get_default_repo() {
+  local ARCH=$1
+  if [[ "$ARCH" == arm* ]]; then
+    echo $DEFAULT_ARM_REPO_URL
+  else
+    echo $DEFAULT_REPO_URL
+  fi
+}
+
+get_core_repo_url() {
+  local REPO_URL=$1 ARCH=$2
+  if [[ "$ARCH" == arm* ]]; then
+    echo "${REPO_URL%/}/$ARCH/core"
+  else
+    echo "${REPO_URL%/}/core/os/$ARCH"
+  fi
+}
+
+get_template_repo_url() {
+  local REPO_URL=$1 ARCH=$2
+  if [[ "$ARCH" == arm* ]]; then
+    echo "${REPO_URL%/}/$ARCH"
+  else
+    echo "${REPO_URL%/}/\$repo/os/$ARCH"
+  fi
+}
+
+configure_pacman() {
+  local DEST=$1 ARCH=$2
+  debug "configure DNS and pacman"
+  cp "/etc/resolv.conf" "$DEST/etc/resolv.conf"
+  SERVER=$(get_template_repo_url "$REPO_URL" "$ARCH")
+  echo "Server = $SERVER" >> "$DEST/etc/pacman.d/mirrorlist"
+}
+
+configure_minimal_system() {
+  local DEST=$1
+
+  mkdir -p "$DEST/dev"
+  echo "root:x:0:0:root:/root:/bin/bash" > "$DEST/etc/passwd"
+  echo 'root:$1$GT9AUpJe$oXANVIjIzcnmOpY07iaGi/:14657::::::' > "$DEST/etc/shadow"
+  touch "$DEST/etc/group"
+  echo "bootstrap" > "$DEST/etc/hostname"
+
+  test -e "$DEST/etc/mtab" || echo "rootfs / rootfs rw 0 0" > "$DEST/etc/mtab"
+  #test -e "$DEST/dev/null" || mknod "$DEST/dev/null" c 1 3
+  #test -e "$DEST/dev/random" || mknod -m 0644 "$DEST/dev/random" c 1 8
+  #test -e "$DEST/dev/urandom" || mknod -m 0644 "$DEST/dev/urandom" c 1 9
+  mount -o bind /dev $DEST/dev
+  mount -o bind /proc $DEST/proc
+
+  sed -i "s/^[[:space:]]*\(CheckSpace\)/# \1/" "$DEST/etc/pacman.conf"
+  sed -i "s/^[[:space:]]*SigLevel[[:space:]]*=.*$/SigLevel = Never/" "$DEST/etc/pacman.conf"
+}
+
+install_rpi_image(){
+  mkdir -p $DEST/root/archlinux_arm
+  wget http://archlinuxarm.org/os/ArchLinuxARM-rpi-2-latest.tar.gz
```

Comment:
as mentionned elsewhere using the arm (32-bits) toolchain for build for aarch64, I don't think this will work.